CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Note that since we don't clearly distinguish between a public and private interf
 
 ## [Unreleased]
 
+## [v3.8.3] - 2022-05-27
+- Add check that a data block contains volume data before parsing
```

Comment:
```suggestion
- Add check that a data block contains volume data before parsing
```

This should be in the unreleased section until we make a new release with the changes.